packages/common/src/test/themeMount.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactElement } from 'react';
+import { mount } from 'enzyme';
+import { ThemeProvider } from 'styled-components';
+import theme from '../theme';
+
+const mountWithTheme = (tree: ReactElement<any>) => {
+  const WrappingThemeProvider: React.FC<{
```

Comment:
Using `React.FunctionComponent` instead of `React.FC` will make it a bit more clear I think